tests/conftest.py

```diff
@@ -142,8 +142,11 @@ def instance(docker_client, profile):
 
     def remove_extra_mounts():
         for extra_mount in instance.profile.extra_mounts:
-            extra_volume, _, _ = instance.profile.parse_extra_mount(extra_mount)
-            docker_client.volumes.get(str(extra_volume)).remove()
```

Comment:
This was a bug in the test suite - we were trying to remove bind mounts as if they were volumes, see the new code below.